src/plugins/dataflow/model/dataflow-content.ts

```diff
@@ -294,10 +294,8 @@ export const DataflowContentModel = TileContentModel
       // Time   |  Node 1 | Node 2 | Node 3 etc
       //    0   |   val    | val    |  val
       addAttributeToDataSet(self.dataSet, { name: "Time (sec)" }); //time quantized to nearest sampling rate
-      let insertionOrder = 1;
```

Comment:
Removed logic when writing to the `dataSet` during Recording, seems redundant to recalculate an insertion order, plus Leslie would like any nodes to match their column names - and this wasn't the case prior because we were using different display names (ex: displayName - Timer (On/Off) vs name: Timer)